solutions/typescript/01-vi6/code/your_program.sh

```
@@ -0,0 +1,15 @@
+#
+# Note: Changing this script WILL NOT affect how CodeCrafters runs your program.
+#
+# Learn more: https://codecrafters.io/program-interface
+
+set -e # Exit early if any commands fail
+
+# Copied from .codecrafters/run.sh
+#
+# - Edit this to change how your program runs locally
+# - Edit .codecrafters/run.sh to change how your program runs remotely
+exec bun run $(dirname $0)/app/main.ts "$@"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)